Basic-Car-Maintenance-Tests/Shared/Models/ContributorTests.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ContributorTests.swift
+//  Basic-Car-Maintenance-Tests
+//
+//  Created by Drag0ndust on 07.10.23.
+//
+
+import XCTest
+@testable import Basic_Car_Maintenance
```

Comment:
Do you know where I could change this somewhere to make the name `Basic-Car-Maintenance` with hyphens rather than underscores?

Reply:
To be honest, no.